ONSdigital/dp-frontend-filter-dataset-controller
# Dataset client replaced

### What

Replaced dataset client with new RCHTTP one

### How to review

* Check the code
* Run the tests
* Run the service

### Who can review

Anyone except me
